lib/llvm-backend/build.rs

```diff
@@ -274,6 +314,8 @@ fn main() {
     println!("cargo:rerun-if-env-changed={}", &*ENV_USE_DEBUG_MSVCRT);
     println!("cargo:rerun-if-env-changed={}", &*ENV_FORCE_FFI);
 
+    install_llvm();
```

Comment:
What happens if LLVM is already downloaded? What happens if there's a suitable LLVM 8.0 installation on the system, but not downloaded by us?

Reply:
I added the if-condition to avoid calling `install_llvm()` when LLVM is already located on the system.